src/WithFileUploads.php

```diff
@@ -102,8 +102,12 @@ protected function dehydratePropertyFromWithFileUploads($name, $value)
     {
         if ($value instanceof TemporaryUploadedFile) {
```

Comment:
Again early returns would be more readable
```php
if ($value instanceof TemporaryUploadedFile) {
    return $value->serializeForLivewireResponse();
}

if (is_array($value) && isset($value[0]) && $value[0] instanceof TemporaryUploadedFile) {
    return $value[0]::serializeMultipleForLivewireResponse($value);
}

if (is_array($value)) {
    foreach (array_keys($value) as $key) {
        $value[$key] = $this->dehydratePropertyFromWithFileUploads(null, $value[$key]);
    }
}

return $value;
```

Reply:
@nuernbergerA - push some updates and thanks for the tip :)